packages/salesforcedx-vscode-core/src/index.ts

```diff
@@ -705,12 +705,12 @@ export async function activate(context: vscode.ExtensionContext) {
     SfdxCommandletExecutor,
     sfdxCoreSettings,
     SfdxWorkspaceChecker,
-    workspaceContext,
+    workspaceContext: workspaceContextInstance,
```

Comment:
This is the one spot where it wasn't clear what the right thing to do is.  `api` is returned by `activate()` and I'm not familiar with how this is used, so I kept it as being returned as, "workspaceContext".  This compiles and all tests pass, and I performed a light smoke test and haven't found any issues.